src/app/app-routing.module.ts

```diff
@@ -19,6 +19,10 @@ const routes: Routes = [
         path: 'products-list', // ['products-list']
         component: ProductsListComponent,
     },
+    {
+        path: 'productCategory/:id',
```

Comment:
Почему решил отдельный путь указать, а не консистентно с `products-list`?
```suggestion
        path: 'products-list/:id',
```

Reply:
У меня не получилось сделать через Children, оказывается просто забыл убрать component перед тем как Children объявлять